utah-design-system-website/src/react/components/websiteContent/library/components/widgetsIndicators/badges/BadgesDocumentation.jsx

```diff
@@ -12,23 +15,63 @@ import {
   TableWrapper,
   TabList,
   TabPanel,
-  TabPanels
+  TabPanels,
+  useBanner,
 } from '@utahdts/utah-design-system';
-import badgesMockup from '../../../../../../../static/images/mockups/Badges.jpg';
-import { LightBox } from '../../../../../lightbox/LightBox';
 import { StaticExample } from '../../../../../staticExamples/StaticExample';
+import { SandboxExample } from '../../../../../sandbox/SandboxExample';
+import { BadgesExampleRender } from './BadgesExampleRender';
+import { BadgesExampleProps } from './BadgesExampleProps';
+import { BadgesExampleCodeReact } from './BadgesExampleCodeReact';
 
 export function BadgesDocumentation() {
+  const { addBanner } = useBanner();
   return (
     <div className="documentation-content">
       <h1 id="h1-top">Badges</h1>
       <p className="lead-in">Badges convey dynamic information, such as counts or status. A badge can include labels or numbers.</p>
       <hr />
       <h2 id="section-example">Example</h2>
+      <SandboxExample
```

Comment:
When the value is a single digit, like the number "3", the circle is narrow and not a circle. Should it have a min width so it is a circle when the content is small?